docs/source/cli/model-training-inference/single-machine-training-inference.rst

```diff
@@ -1,6 +1,74 @@
 .. _single-machine-training-inference:
 
-Model Training and Inference on a Single Machine
+Model Training and Inference in a Single Machine
```

Comment:
on?

Reply:
Let's stick on `on` for the documentation.